js/aax.js

```diff
@@ -2392,6 +2392,60 @@ module.exports = class aax extends Exchange {
         return tiers;
     }
 
+    async transfer (code, amount, fromAccount, toAccount, params = {}) {
+        await this.loadMarkets ();
+        const currency = this.currency (code);
+        const accountTypes = this.safeValue (this.options, 'types', {});
+        const fromId = this.safeString (accountTypes, fromAccount);
+        const toId = this.safeString (accountTypes, toAccount);
+        if (fromId === undefined) {
+            const keys = Object.keys (accountTypes);
+            throw new ExchangeError (this.id + ' fromAccount must be one of ' + keys.join (', '));
+        }
+        if (toId === undefined) {
+            const keys = Object.keys (accountTypes);
+            throw new ExchangeError (this.id + ' toAccount must be one of ' + keys.join (', '));
+        }
+        const request = {
+            'currency': currency['id'],
+            'fromPurse': fromId,
+            'toPurse': toId,
+            'quantity': amount,
+        };
+        const response = await this.privatePostAccountTransfer (this.extend (request, params));
+        //
```

Comment:
Let's move lines 2416-2446 to `parseTransfer` (see mentions of `parseTransfer` in the other exchanges).